config/routes.rb

```diff
@@ -24,9 +24,13 @@
 
   resources :myprojects
 
+  match "/myprojects/:id/edit", to: "myprojects#update",  :via => [:get, :put]
```

Comment:
Please remove this line it does not solve the issue.
`form_tag {action: :update}`